scripts/deploy.sh

```diff
@@ -56,12 +56,15 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   export DEPLOY_HAPROXY="yes"
   if [[ ! -d /etc/openstack_deploy/ ]]; then
     ./scripts/bootstrap-aio.sh
+    # remove unused variables file
+    rm /etc/openstack_deploy/user_variables.yml
```

Comment:
Removing this file means that the OSA AIO defaults, currently being used by RPCO AIO's, are no longer set. The vars are defined in the OSA file 'tests/roles/bootstrap-host/templates/user_variables.aio.yml.j2'. How have you taken this into account?